lib/rewrite/rewrite-expr-grammar.ym

```diff
@@ -106,6 +106,11 @@ rewrite_expr
           }
           '(' rewrite_set_opts ')'              { $$ = last_rewrite; }
 	| KW_SET_TAG '(' string ')'             { $$ = log_rewrite_set_tag_new($3, TRUE, configuration); free($3); }
+	| KW_SET_TAG '(' string 
```

Comment:
This works as expected, good job!

To further improve it, the two `KW_SET_TAG` rules (this one, and the one above) could be merged.
`KW_GROUP_SET` shows an example with `rewrite_groupset_opts`:
```
rewrite_groupset_opts
        : rewrite_groupset_opt rewrite_groupset_opts
        |
        ;
```
We use this pattern in nearly every grammar file. It allows setting multiple options multiple times, and the empty `|` also supports not setting any option, so this is kind of a non-mandatory option group.

Reply:
Hi @alltilla,

Thanks for taking the time to review my PR. I have made changes according to your suggestion.